store.go

```diff
@@ -46,7 +46,7 @@ func setStoreFormatFromData(data []byte) {
 		StoreFormat = YAML
 		return
 	}
-	log.Fatalf("Unable to determine format from content")
+	log.Printf("Unable to determine format from content")
```

Comment:
I'm thinking if we revert this back to Fatalf and update readJSONData to be:
```go
func ReadJSONData(data []byte) GossConfig {
    setStoreFormatFromData(data)
    if StoreFormat == UNSET {
        setStoreFormatFromData(data)
    }
   ...
```

The user will get clear error messages in all scenarios, what do you think?